omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js

```diff
@@ -51,10 +51,6 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
             self.saveCondition();
         }
 
-    self.unselectStatus = self.unselectStatus || function () {
```

Comment:
Is this part of the ticket?

Reply:
Yes, This is to remove the functionality of cross button 'x'  as mentioned on the ticket